kubeflow/core/prototypes/tensorboard.jsonnet

```diff
@@ -0,0 +1,14 @@
+// from github.com/jlewi/kubeflow-rl
+
```

Comment:
You should have a header like in TFServing in which you provide a description and parameters
https://github.com/kubeflow/kubeflow/blob/master/kubeflow/tf-serving/prototypes/tf-serving-all-features.jsonnet